preprocessors/openstreetmap/main.py

```diff
@@ -54,26 +65,58 @@ def get_map_data():
 
     if validated is not None:
```

Comment:
You are checking for validated before it is declared! Surprised this didn't produce an error unless I am misinterpreting this. Do you need this check here?

Reply:
It was declared before being called. You can actually find the function in the ```osm_service.py``` file (line 973) and imported into the ```main.py`` file (line 15).